content/welcome/_index.en.md

```diff
@@ -32,17 +38,16 @@ It's important to start R every time with a clean working session. To do this, y
 
 Your Global Options window should look like this:
 
-<img src="https://imgur.com/LXsbqPT.png" width = "50%" align="middle" alt="RStudio global options window showing a setting optimal for reproducibility">
+![RStudio global options window showing a setting optimal for reproducibility](images/RStudio_global_options.jfif?width=30pc)
```

Comment:
You might want to change this file name. jfif is not a commonly recognized format.